CHANGELOG.md

```diff
@@ -9,7 +9,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
 
 - **Hero**
-  - Add a hero components `hero` ([ce2c71d](https://github.com/studiometa/ui/commit/ce2c71d)
+
```

Comment:
**nitpick:** Remove the empty lines between each item of the list

You can see the rendering difference here: https://github.com/studiometa/ui/blob/feature/hero-component/CHANGELOG.md